### PR TITLE
add precurse

### PR DESCRIPTION
**Pull request by LLorch1984**

### CODE REVIEW

File: staff/jorge-genove/precourse/bingo-pro/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+var cartonFake = [];
+var carton = [];
+var randomNumberCheck = [];
+var user;
+var cartonControl = [100];
+var lineaCheck = false;
+var controlName = false;
+var stop = false;
+function impCarton() {
+  for (let i = 0; i < carton.length; i++) {
```
Comment (sergicast) [inline staff/jorge-genove/precourse/bingo-pro/index.js:16]: change let for var

Comment (Aitor1111) [inline staff/jorge-genove/precourse/bingo-pro/index.js:16]: it seems to be a `let` declaration? OMG dude this not ES5

Comment (LLorch1984) [reply staff/jorge-genove/precourse/bingo-pro/index.js]: done

Comment (LLorch1984) [reply staff/jorge-genove/precourse/bingo-pro/index.js]: done

**Pull request merged**